proxy/src/main/java/com/velocitypowered/proxy/scheduler/VelocityScheduler.java

```diff
@@ -65,10 +68,28 @@ public VelocityScheduler(PluginManager pluginManager) {
 
   @Override
   public TaskBuilder buildTask(Object plugin, Runnable runnable) {
-    checkNotNull(plugin, "plugin");
-    checkNotNull(runnable, "runnable");
+    requireNonNull(runnable, () -> "the scheduled task cannot be null");
```

Comment:
why change this?

Reply:
I thought it would be better to use a Supplier to avoid creating an unnecessary string each time this method is called and no argument is null

Reply:
The string is stored in the constant pool. There is no need to "create an unnecessary string".